src/panel/poi/PoiPanel.jsx

```diff
@@ -233,7 +234,7 @@ export default class PoiPanel extends React.Component {
         text: _('Back to favorite'),
         className: 'poi_panel__back_to_list',
       };
-    } else if (isFromCategory) {
+    } else if (resource && (resource.category || resource.query)) {
```

Comment:
You can put a default prop value for `resource` in this component too so you can remove the existence tests everywhere and have shorter lines (the next one breaks linter rules)

Reply:
Yep, I did this in `CategoryPanel.jsx ` already but missed this one, thanks